apps/apps-backend/src/features/features.controller.ts

```diff
@@ -28,9 +28,23 @@ export class FeaturesController {
                 [Feature.WalletSentryTracing]: {
                     defaultValue: 0.0025,
                 },
-                // Note: we'll add wallet dapps when evm will be ready
                 [Feature.WalletDapps]: {
-                    defaultValue: [],
+                    defaultValue: [
+                        {
+                            name: 'Wallet Dashboard',
+                            description: 'Staging version of the Wallet Dashboard',
```

Comment:
also, is this description something that should be displayed? i'm not sure what its purpose is :thinking: :pray: